ingestors/support/convert.py

```diff
@@ -35,43 +39,53 @@ def document_to_pdf(self, file_path, entity):
             self.tags.set(key, content_hash)
         return pdf_file
 
-    def _document_to_pdf(self, file_path, entity):
+    def _document_to_pdf(self, file_path, entity, timeout=TIMEOUT):
         """Converts an office document to PDF."""
         file_name = entity_filename(entity)
-        mime_type = entity.first("mimeType")
         log.info("Converting [%s] to PDF", entity)
-        for attempt in range(1, CONVERT_RETRIES + 1):
-            try:
-                with open(file_path, "rb") as fh:
-                    files = {"file": (file_name, fh, mime_type)}
-                    res = requests.post(
-                        explicit_resolve(CONVERT_URL),
-                        params={"timeout": CONVERT_TIMEOUT},
-                        files=files,
-                        timeout=CONVERT_TIMEOUT + 10,
-                        stream=True,
+
+        pathlib.Path(OUT_DIR).mkdir(parents=True, exist_ok=True)
+
+        cmd = [
+            "/usr/bin/libreoffice",
+            '"-env:UserInstallation=file:///tmp"',
+            "--nologo",
+            "--headless",
+            "--nocrashreport",
```

Comment:
Is there a world in which we'd want to try and capture this information in the future?